src/api-fetch-setup.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import { fetchRequest } from 'react-native-gutenberg-bridge';
```

Comment:
Is this from an older attempt or I’m missing where it’s used?

Reply:
Yes, this was part of the tests making network from the parent apps. I plan to post a PR with this other approach.